uaclient/cli.py

```diff
@@ -241,7 +241,8 @@ def action_attach(args, cfg):
         raise exceptions.NonRootUserError()
     contract_client = contract.UAContractClient(cfg)
     if not args.token:
-        bound_macaroon_bytes = sso.discharge_root_macaroon(contract_client)
+        with util.disable_log_to_console():
```

Comment:
This is the context manager call that I think we can avoid  now that you do that in cli.py

Reply:
invalid, as you mentioned in IRC. we aren't handling SSOAuth errors etc in : https://github.com/CanonicalLtd/ubuntu-advantage-client/blob/master/uaclient/sso.py#L172-L176. So this stands